src/Sylius/Bundle/OrderBundle/Factory/OrderItemUnitFactoryInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Sylius\Bundle\OrderBundle\Factory;
+
+use Sylius\Component\Order\Model\OrderItemInterface;
+use Sylius\Component\Order\Model\OrderItemUnit;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface OrderItemUnitFactoryInterface extends FactoryInterface
```

Comment:
IMHO it shouldn't extend `FactoryInterface` as its only implementation throws "unsupported method" exception and the other ones won't be supporting `createNew()` too.